prombench/cmd/prombench/Dockerfile

```diff
@@ -9,7 +9,10 @@ ENV TEST_INFRA_REPO https://github.com/prometheus/prombench.git
 
 ENV PROMBENCH_DIR $TEST_INFRA_DIR/prombench
 
-COPY prombench /usr/bin/prombench
-COPY cmd/prombench/clone.sh /usr/bin/clone.sh
+ARG ARCH="amd64"
+ARG OS="linux"
+
+COPY .build/${OS}-${ARCH}/prombench /usr/bin/prombench
```

Comment:
I think you need to fix `.promu.yml` as the binary is stored at `.build/${OS}-${ARCH}/prombench/prombench` currently.

https://github.com/prometheus/prombench/blob/4ca81fe3e10c43f44ab65d468158c946032743bf/.promu.yml#L9

It should be `name: prombench`

Reply:
@simonpasquier this will mean there will be no local docker builds using `Makefile.common` right?

Reply:
Not sure what you mean by that.

Reply:
oh sorry. I though https://github.com/prometheus/prometheus/pull/5609 was still persisting.